### ISSUE TITLE
cut a new release plz?

### ISSUE THREAD

**Issue by qmx** [sfackler/cargo-tree#30]
currently `cargo install cargo-tree` fails with problems building `cargo-0.22.0`. This is fixed on master by bb2040084947d5d6063d15c8e97d482bea125aeb

you can repro it with a simple dockerfile:
```
FROM rust:1.22.1
RUN apt-get update && apt-get install -y cmake
RUN cargo install cargo-tree --vers=0.15.0
```

Thanks!

**Comment by sfackler** [comment]
I'll cut a new release, but 0.15.0 seems to build for me just fine in rust:1.22.1 - what errors were you seeing?

**Comment by qmx** [comment]
looks like it was a transient error, I've managed to build just fine now.

for posterity, that was the error:

```
Compiling cargo v0.22.0
error[E0277]: the trait bound `for<'de> Flags: serde::Deserialize<'de>` is not satisfied
   --> usr/local/cargo/registry/src/github.com-1ecc6299db9ec823/cargo-0.22.0/src/cargo/lib.rs:123:24
    |
123 |     let flags = docopt.deserialize().map_err(|e| {
    |                        ^^^^^^^^^^^ the trait `for<'de> serde::Deserialize<'de>` is not implemented for `Flags`
    |
    = help: consider adding a `where for<'de> Flags: serde::Deserialize<'de>` bound
    = note: required because of the requirements on the impl of `serde::de::DeserializeOwned` for `Flags`

error: aborting due to previous error

error: failed to compile `cargo-tree v0.15.0`, intermediate artifacts can be found at `/tmp/cargo-install.YaGywC3AWrLS`

Caused by:
  Could not compile `cargo`.

To learn more, run the command again with --verbose.
The command '/bin/sh -c cargo install cargo-tree --vers=0.15.0' returned a non-zero code: 101

```

**Comment by sfackler** [comment]
Released 0.16
